buku.py

```diff
@@ -4095,6 +4113,8 @@ def main():
                          special keywords -
                          "blank": entries with empty title/tag
                          "immutable": entries with locked title
+    -W, --without        combine with keyword search to exclude
```

Comment:
Now that each option has it's own keywords, please specify [...] after `sany`, `sall`, `sreg` and `without` (like we have after `--title`).

Reply:
When we add a new option it goes to documentation as well (man, readme) but I will cover it in this case. We are already lagging in documentation in certain other areas.

Reply:
![2018-03-25 12_17_23-cmd](https://user-images.githubusercontent.com/6178847/37877285-b9d62e56-3026-11e8-81ca-6c7239eb3b95.png)

See attached screenshot for confirmation.